### PR TITLE
IDE-1866 add the suspend validations setting file for sdk Project

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

File: tools/plugins/com.liferay.ide.sdk.core/src/com/liferay/ide/sdk/core/SDKUtil.java [hunk 1]
```diff
@@ -266,6 +269,26 @@ public static void openAsProject( SDK sdk )
...
+
+                if( !settingFolderPath.append( "org.eclipse.wst.validation.prefs" ).toFile().exists() )
+                {
+                    URL url =
+                        FileLocator.toFileURL( SDKCorePlugin.getDefault().getBundle().getEntry(
+                            "files/org.eclipse.wst.validation.prefs" ) );
+
+                    File file = new File( url.getFile() );
+
+                    FileUtil.copyFileToDir( file, settingFolder );
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.sdk.core/src/com/liferay/ide/sdk/core/SDKUtil.java]: Lets only copy this if the original file doesn't exist.

**Comment by gamerson** [comment]
Closing see comments and resend.

File: tools/plugins/com.liferay.ide.sdk.core/src/com/liferay/ide/sdk/core/SDKUtil.java (hunk 1 above)
Comment (jtydhr88) [inline tools/plugins/com.liferay.ide.sdk.core/src/com/liferay/ide/sdk/core/SDKUtil.java]: Hey Greg, this code would check the original validation prefs file existing or not.

Comment (gamerson) [inline tools/plugins/com.liferay.ide.sdk.core/src/com/liferay/ide/sdk/core/SDKUtil.java]: I see it now, thanks

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Merged, thanks! See f3cc4af...22169b8
f3cc4af...22169b8 <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**